docs: Use HTTPS for CoC (#776)
EddieHubCommunity/hacktoberfest-practice: CONTRIBUTING.md

# Contributing

We love pull requests from everyone. By participating in this project, you
agree to abide by the [Code Of Conduct](https://github.com/EddieHubCommunity/.github/blob/main/CODE_OF_CONDUCT.md).

> 

Reviewer: ```suggestion
```

# Contributing

We love pull requests from everyone. By participating in this project, you

> agree to abide by the [Contributor Covenant](https://www.contributor-covenant.org/).

Reviewer: ```suggestion
agree to abide by the [Code Of Conduct](https://github.com/EddieHubCommunity/.github/blob/main/CODE_OF_CONDUCT.md).
```

> 

Reviewer: ```suggestion
```

# Contributing

We love pull requests from everyone. By participating in this project, you
agree to abide by the [Code Of Conduct](https://github.com/EddieHubCommunity/.github/blob/main/CODE_OF_CONDUCT.md).

> ## Steps

Reviewer: idk what happened but you didn't had to remove both the lines 😅 

```suggestion

## Steps
```

Author: ooof, this might be my most confusing PR yet 🤦 , lemme know about any more changes (which I'm sure there are)

1. [Fork](https://help.github.com/articles/fork-a-repo/) this project
2. [Clone](https://help.github.com/articles/fork-a-repo/#step-2-create-a-local-clone-of-your-fork) your forked version `git clone git@github.com:<YOUR-USERNAME>/hacktoberfest-practice.git`
3. Make changes

> 4. Agree to abide by the [Contributor Covenant](https://www.contributor-covenant.org/).

Reviewer 2: ```suggestion
4. [Commit](https://help.github.com/articles/adding-a-file-to-a-repository-using-the-command-line/) your changes (write a short descriptive message of what you have done)
```

Author: wait, this is literally the older version I guess
